ush/setup.py

```diff
@@ -75,6 +75,9 @@ def load_config_for_setup(ushdir, default_config_path, user_config_path):
     )
     logging.debug(user_config)
 
+    machine = user_config["user"]["MACHINE"].upper()
```

Comment:
@mkavulich Is it possible to do this on one line, like:
```
    user_config["user"]["MACHINE"] = user_config["user"]["MACHINE"].upper()
```

Reply:
The assignment is for both `machine` (local variable) and `user_config["user"]["MACHINE"]` (config variable), so two different assignments are necessary. This is previously existing logic that was moved up.

Reply:
Ok, makes sense.